### PR TITLE
refactor: unattended upgrade without go template

### PR DESCRIPTION
**Pull request by alexeldeib**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by UtheMan — APPROVED** [review]

File: parts/linux/cloud-init/artifacts/cse_main.sh [hunk 1]
```diff
@@ -291,17 +291,16 @@ if $REBOOTREQUIRED; then
...
-        systemctl restart --no-block apt-daily.service
+        if [ "${ENABLE_UNATTENDED_UPGRADES}" == "true" ]; then
+            systemctl unmask apt-daily.service apt-daily-upgrade.service
+            systemctl enable apt-daily.service apt-daily-upgrade.service
+            systemctl enable apt-daily.timer apt-daily-upgrade.timer
+            systemctl restart --no-block apt-daily.timer apt-daily-upgrade.timer            
+            # this is the DOWNLOAD service
+            # meaning we are wasting IO without even triggering an upgrade 
+            # -________________-
+            systemctl restart --no-block apt-daily.service
```
Comment (cameronmeissner) [inline parts/linux/cloud-init/artifacts/cse_main.sh:302]: looks like we were doing this outside the `if EnableUnattendedUpgrade` block?

Comment (alexeldeib) [reply parts/linux/cloud-init/artifacts/cse_main.sh:302]: that was a bug, restarting a masked service does nothing

**Review by ganeshkumar5699 — COMMENTED** [review]
Does nodecustomdata.yml also need to be changed?

**Comment by alexeldeib** [comment]
> Does nodecustomdata.yml also need to be changed?

eventually

image

**Pull request merged**